atlp-rwanda/e-commerce-lydia-32-bn
# Ft view cart 187570884 Buyer should be able to view their cart

### What does this PR do?

This PR implements the functionality for the buyer to view their cart. It retrieves the cart data from the buyer's session, fetches the product details from the database for each item in the cart, calculates the cart total, and sends the cart data to the frontend to be displayed to the buyer.
### Description of Task to be completed?

The task is to enable buyers to view their shopping cart, including the items they have added, the product details (name, price, image), and the total cost of the items in the cart.

### How should this be manually tested?

- Start the application and log in as a buyer.
- Add some products to the cart.
- Navigate to the cart route
- Verify that the cart displays all the items you added, along with their respective product details (name, price, image).
- Ensure that the cart total is correctly calculated and displayed.

### What are the relevant pivotal tracker stories
  https://www.pivotaltracker.com/n/projects/2703469
